docker/start_s3rver.sh

```diff
@@ -1,3 +1,10 @@
 #!/bin/bash
 
-make start-s3rver > /dev/null &
+# exit if s3rver is already running
```

Comment:
```suggestion
# check if lsof has been installed
if ! command -v lsof > /dev/null ; then
    echo "For faster start-up in the future, update the PrairieLearn image using:"
    echo -e "\tdocker pull prairielearn/prairielearn"
    yum install -y lsof
fi

# exit if s3rver is already running
```

Reply:
This could be a temporary change, and could be removed in a few weeks or months.

Reply:
I ended up going with the "die" version to make sure people upgrade immediately 😄

Reply:
Just tested, and it works fine here. The irony is that `docker pull` won't be a solution until it is merged, so if I want to test properly I have to do something else instead in the meantime...

Reply:
Haha, yes, that's an unfortunate irony!